Frontend/src/hooks/eventListener.ts

```diff
@@ -2,7 +2,9 @@ import { useCallback, useEffect, useRef } from 'react';
 
 export function useEventListener(
   eventType: string,
+  // eslint-disable-next-line
   onEventTrigger: React.EventHandler<any>,
+  // eslint-disable-next-line
   dependents: any[]
```

Comment:
`dependents` er ikke i bruk her, så den kan vi fjerne (og fjerne fra der funksjonen kalles), og `any` mener jeg kan erstattes slik (men sikkert lurt å verifisere at det funker 😅 ):

```suggestion
export function useEventListener<K extends keyof GlobalEventHandlersEventMap>(
  eventType: K,
  onEventTrigger: (event: GlobalEventHandlersEventMap[K]) => void,
```

Reply:
Så alt dette funker!
Men når jeg undersøkte sletting av dependents så ser jeg at ingenting av dette egentlig brukes.
Stedet der det brukes på, brukes ikke.

Så jeg sletter liksågreit alt sammen.

Nydelig løsning da, det fungerte 🤩

Reply:
Oooh, digg! 🪓